test/source/tests/decrypt.ts

```diff
@@ -892,6 +892,43 @@ XZ8r4OC6sguP/yozWlkG+7dDxsgKQVBENeG6Lw==
       })
     );
 
+    test(
+      'signature - cleartext signed messages from HTML are re-fetched when needed',
+      testWithBrowser('ci.tests.gmail', async (t, browser) => {
+        const acctEmail = 'ci.tests.gmail@flowcrypt.test';
+        const settingsPage = await browser.newExtensionSettingsPage(t, acctEmail);
+        const accessToken = await BrowserRecipe.getGoogleAccessToken(settingsPage, acctEmail); // todo: include in t?
```

Comment:
I think we can simplify some tests if we can extract google access token from `t` -- we can put it there in a helper method like `setUpCommonAcct`, so we don't have to open a dummy page just to retrieve the access token.

Reply:
Yeah, it'll be nice to have access token available through `t`. Probably we can include such improvement when working on part 2 of #4494, there we can fetch access token when configuring test account.

Reply:
#4966